style.css

```diff
@@ -148,6 +148,428 @@ body {
   cursor: pointer;
 }
 
+.middle-section {
+  min-height: 281px;
+  padding-bottom: 51px;
+  background: #fae8e4;
+}
+
+.middle-section__image {
+  height: 264px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  overflow: hidden;
+  margin-top: 62px;
+  padding: 0;
+}
+
+.middle-section__img {
+  margin-top: 14px;
+}
+
+.middle-section__img img {
+  width: 100%;
+  height: 100%;
+  -o-object-fit: contain;
+  object-fit: contain;
+}
+
+.middle-section__clutch {
+  padding: 15px 30px;
+  margin: 0 28px 0;
+  background: #fff;
+  -webkit-box-shadow: 11px 25px 40px -34px rgba(0,0,0,.8);
+  box-shadow: 11px 25px 40px -34px rgba(0,0,0,.8);
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5835770)